_rules/element-lang-matches-default-off6ek.md

```diff
@@ -0,0 +1,339 @@
+
+- the element is an [inclusive descendant][] in the [flat tree][] of a `body` element; and
+- The element is in a [top-level browsing context][]; and
+- The element is in a [document][] with a [content type][] of `text/html`; and
+- The element has a `lang` [attribute value][] which is a [valid language tag][]; and
+- The element has a defined [default element language][].
+
+## Expectation
+
+For each test target, the [primary language][] of the [valid language tag][] matches its [default element language][].
+
+## Assumptions
+
+- This rule assumes that the default human language of a page, as described in WCAG 2, can be determined by counting the number of words used in each language. If the default language needs to be derived in some other way (such as frequency analysis, mutual information based distance, …), this rule may fail while [Success Criterion 3.1.2 Language of Parts][sc312] is still satisfied.
+
+- This rule assumes that user agents and assistive technologies can programmatically determine [valid language tags][] even if these do not conform to the [BCP 47][] syntax.
+
+- This rule assumes that [grandfathered tags][] are not used as these will not be recognized as [valid language tags][].
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+This rule ignores elements that are nested inside `iframe`. `iframe` are intended to provide a layer of isolation, and it is not clear how language is inherited in nested browsing contexts. Incorrect `lang` attribute inside `iframe` are still likely to cause accessibility issues even though this rule won't flag them.
+
+- Related rule:
+  - [_Element with `lang` Attribute Has Valid Language Tag_](https://act-rules.github.io/rules/de46e4)
+- [Understanding Success Criterion 3.1.2: Language of Page][usc312]
+- [H58: Using language attributes to identify changes in the human language](https://www.w3.org/WAI/WCAG21/Techniques/html/H58)
+- [BCP 47: Tags for Identifying Languages](https://www.ietf.org/rfc/bcp/bcp47.txt)
+- [The `lang` and `xml:lang` attributes](https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes)
+
+In all examples, the `html` element has itself a `lang` attribute in order to make sure that the examples do not fail [Success Criterion 3.1.1 Language of Page](https://www.w3.org/TR/WCAG21/#language-of-page). These `html` elements are, however, never applicable because they are not descendants of a `body` element, and the example descriptions do not mention them further.
+
+Similarly, all examples contain a short text in the page language without any `lang` attribute in order to make the `lang` attribute on other elements useful. Especially, Technique [H58: Using language attributes to identify changes in the human language](https://www.w3.org/WAI/WCAG21/Techniques/html/H58) is about changes of language.
+
+## Test Cases
```

Comment:
Examples feel fairly contrived with nested `lang` attributes… Not sure it's possible to do much better given the nature of the rule 🤷 

Especially tried to make sure that all targets in a given example pass/fail the rule together to avoid mixed results and make tool validation easier.